src/mmw/js/src/compare/templates/compareScenarios.html

```diff
@@ -0,0 +1,7 @@
+<div id="edge-fade"></div><!-- Blur offcanvas scenario -->
+
+<table> <!-- Table allows for horizontal scrolling. -->
```

Comment:
We don't want horizontal scrolling because each scenario will have vertical scrolling in the modification list.  It gets to be too many scrollbars and hard to use.  I could see a use case for being able to swipe to scroll horizontally on touch.  Is this something we can enable just for touch?